File: go/models/News.go
package models

import "github.com/go-pg/pg"

type TblNews struct {
	tableName struct{} `sql:"tbl_news,alias:t" pg:",discard_unknown_columns"`

	ID                string      `sql:"id,pk"`
	Content           string      `sql:"content,notnull"`
	ContentFromScrapy *string     `sql:"content_from_scrapy"`
	CreateTime        pg.NullTime `sql:"create_time"`
	DeleteTime        pg.NullTime `sql:"delete_time"`
	DiscussNum        *int        `sql:"discuss_num"`
	LikedNum          *int        `sql:"liked_num"`
	Md5               *string     `sql:"md5"`
	MediaImgs         []string    `sql:"media_imgs,array"`
	MediaTitle        string      `sql:"media_title,notnull"`
	MediaType         *string     `sql:"media_type"`
	NewsUrl           *string     `sql:"news_url"`
	Precontent        string      `sql:"precontent,notnull"`
	RepostdNum        *int        `sql:"repostd_num"`
	UserID            string      `sql:"user_id,notnull"`
}

File: src/main/java/com/shumsg/model/entity/News.java
package com.shumsg.model.entity;

import lombok.Data;

import java.time.LocalDateTime;
import java.util.Date;

@Data
public class News {
    private String id;

    private String userId;

    private String content;

    private String preContent;

    private String mediaType;

    private Object mediaImgs;

    private String mediaTitle;

    private String newsUrl;

    private Object newsLabels;

    private Integer discussNum;

    private Integer discussLikedNum;

    private Integer likedNum;

    private Integer rePostNum;

    private Integer viewsNum;

    private String contentFromScrapy;

    private String md5;

    private LocalDateTime createTime;

    private LocalDateTime discussUpdateTime;

    private LocalDateTime deleteTime;
}